src/ucp/core/ucp_context.h

```diff
@@ -509,6 +509,15 @@ ucp_context_dev_idx_tl_bitmap(ucp_context_h context, ucp_rsc_index_t dev_idx,
                               ucp_tl_bitmap_t *tl_bitmap);
 
 
+void ucp_tl_bitmap_validate(const ucp_tl_bitmap_t *tl_bitmap,
+                            const ucp_tl_bitmap_t *tl_bitmap_super);
+
+
+void ucp_tl_bitmap_validate_dev_name(const ucp_tl_bitmap_t *tl_bitmap,
```

Comment:
seems this func used only once, so i'd revert it to be part of ucp_ep_create_to_worker_addr